kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -302,41 +326,63 @@ define([
 
         function validateFloat(value, min, max) {
             if (isNaN(value)) {
-                return 'value must be numeric';
+                return {
```

Comment:
This was also present in the `validators/float` file. Not sure if it's an oversight that `validateInt` doesn't have it, or intentional. Kinda moot, anyway, since it should only get a Number as input.